packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -264,13 +264,13 @@ html:global(.is-navInView) {
         background-color: $color-support-brand-01;
 
         @include media('>mid') {
-            min-height: 88px;
+            min-height: 80px;
```

Comment:
might be worth moving it to the variable

Reply:
thanks @1itvinka , using the `$header-height` variable here now instead 👍